driver/src/main/java/oracle/nosql/driver/http/Client.java

```diff
@@ -644,6 +647,8 @@ public Result execute(Request kvRequest) {
                                        responseHandler.getHeaders(),
                                        wireContent,
                                        kvRequest);
+                rateDelayedMs += getRateDelayedFromHeader(
```

Comment:
This amount is passed up to the Result object which has an accessor. That method does not have an @since tag on it. Can that be added? Are there other public interfaces that have been added related to rate limiting that need @since?